frontend/views/containers/chatroom/ChatMembersAllModal.vue

```diff
@@ -212,18 +233,44 @@ export default ({
       this.initializeMembers()
     },
     initializeMembers () {
-      const members = this.isJoined ? this.chatRoomUsersInSort : this.details.participants
-      this.addedMembers = members.map(member => ({ ...member, departedDate: null }))
-      this.canAddMembers = this.groupMembersSorted
-        .filter(member => !this.addedMembers.find(mb => mb.username === member.username) && !member.invitedBy)
-        .map(member => ({
-          username: member.username,
-          displayName: member.displayName,
-          joinedDate: null
-        }))
+      if (this.isDirectMessage()) {
+        this.addedMembers = Object.keys(this.chatRoomUsers)
+          .map(username => {
+            return username === this.ourUsername
+              ? {
+                  displayName: this.globalProfile(username).displayName,
+                  username,
+                  departedDate: null
+                }
+              : {
+                  displayName: this.ourContactProfiles[username].displayName,
+                  username,
+                  departedDate: null
+                }
```

Comment:
@Silver-IT Not a big deal but looks like the return block here can be simplified as
`return { username, departedDate: null, displayName: condition ? value1 : value2 }`

Reply:
Correct, we can simplify. For better understanding, I am doing this way.
```
const profile = username === this.ourUsername ? this.globalProfile(username) : this.ourContactProfiles[username]
return {
  displayName: profile.displayName,
  username,
  departedDate: null
}
```